Support/bone101/UI/bbui.js

```diff
@@ -921,8 +1065,8 @@ var UI = (function() {
                 coords[0] = event.clientX + document.body.scrollLeft + document.documentElement.scrollLeft;
                 coords[1] = event.clientY + document.body.scrollTop + document.documentElement.scrollTop;
             }
-            //console.log("Position = " + [x, y]);
-            //console.log("Offset = " + [rect.left, rect.top]);
+            // console.log("Position = " + [x, y]);
```

Comment:
This happens automatically because I'm using beautify tool in sublime text
editor :)

On 13 June 2016 at 15:35, Jason Kridner <notifications@github.com> wrote:

> In Support/bone101/UI/bbui.js
> <https://github.com/jadonk/bone101/pull/39#discussion_r66791810>:
>
> > @@ -921,8 +1065,8 @@ var UI = (function() {
> >                  coords[0] = event.clientX + document.body.scrollLeft + document.documentElement.scrollLeft;
> >                  coords[1] = event.clientY + document.body.scrollTop + document.documentElement.scrollTop;
> >              }
> > -            //console.log("Position = " + [x, y]);
> > -            //console.log("Offset = " + [rect.left, rect.top]);
> > +            // console.log("Position = " + [x, y]);
>
> What is the point of this delta?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jadonk/bone101/pull/39/files/3c6ed84f23c489652aebe3f4807f50d5c5db3333#r66791810>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AG1g0mEum9as53waPtjDQCiWr8QC_lSdks5qLVySgaJpZM4IxAls>
> .
>